ckanext/lacounts/harvest/processors/socrata.py

```diff
@@ -1,2 +1,23 @@
+import logging
+from ckanext.lacounts.harvest import helpers
+log = logging.getLogger(__name__)
+
+
 def socrata_processor(package, harvest_object):
+
+    # Pre-map
+    package['harvest_dataset_url'] = package.get('url')
+
+    # Map
+    package = helpers.map_package(package, {
+        # General
+        'metadata_created': ['source_created_at'],
```

Comment:
We should not change `metadata_modifed` and `metadata_created` as this are internal CKAN fields that refer to our records (eg when a dataset was harvested). We need two new fields in the dataset schema if they are not there: `issued` and `modified` and that's where we should store these values.

Same for all harvesters (ckan and esri)